docs/modules/nifi/pages/usage_guide/security.adoc

```diff
@@ -109,19 +114,21 @@ With this authorization method, a single user has administrator capabilities.
 [#authorization-ldap]
 === LDAP
 
-The operator uses the https://nifi.apache.org/docs/nifi-docs/html/administration-guide.html#fileusergroupprovider[`FileUserGroupProvider`] and https://nifi.apache.org/docs/nifi-docs/html/administration-guide.html#fileaccesspolicyprovider[FileAccessPolicyProvider] to bind the LDAP user to the NiFi administrator group. This user is then able to create and modify groups and polices in the web interface. These changes local to the `Pod` running NiFi and are *not* persistent.
+The operator uses the {nifi-docs-fileusergroupprovider}[`FileUserGroupProvider`] and {nifi-docs-fileaccesspolicyprovider}[FileAccessPolicyProvider] to bind the LDAP user to the NiFi administrator group.
+This user is then able to create and modify groups and polices in the web interface.
```

Comment:
```suggestion
This user is then able to create and modify groups and policies in the web interface.
```

Reply:
Random thought, what is the NiFi API like? Can we create new CRDs for the operator to manage things like this?